learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 2 Learning Outcomes
+
+## HTTP
+
+### Asynchronicity
+
+- [ ] **We can write code that executes asynchronously**
+- [ ] We can use callbacks to make our code non blocking
```

Comment:
The callback itself doesn't make code non-blocking, it just allows us to access a value later on from code that got pushed to the back of the queue. Also we should have promises here if they're going to be using fetch

```suggestion
- [ ] We can use callbacks to access values that aren't available synchronously
- [ ] We can use promises to access values that aren't available synchronously
```
```